### PR TITLE
clipper: update checksum for v4.10.0 

### LINKED ISSUE #12064: [package] clipper/4.10.0: Error in source() method
### Package and Environment Details

* Package Name/Version: **clipper/4.10.0**
* Operating System+version: **Linux Ubuntu 20.04.4 LTS** and **Microsoft Windows Server 2022 10.0.20348 Datacenter**
* Compiler+version: **Clang 13**
* Conan version: **conan 1.51.0**

### Conan profile

I use the default profile.

### Steps to reproduce

1. My [Conanfile](https://github.com/ShenMian/Graphics/blob/78eef3b1d97a375a1644dad51fdc94b3d98349a4/conanfile.txt)->assimp->cliper.
2. Run [script](https://github.com/ShenMian/Graphics/blob/3a82c2d0c0c8b98c8218b464ea3eaaf86b7fae7d/Scripts/build.sh), [Full github workflow file](https://github.com/ShenMian/Graphics/blob/bd8708e667df5cf7c3d3a0ba511852ae2f369f17/.github/workflows/ubuntu.yml).

I have tried to run the workflow again, but the problem still exists. This problem has not occurred before.  

### Logs

<details><summary>Click to expand log</summary>

Ubuntu:  

```
WARN: Remotes registry file missing, creating default one in /home/runner/.conan/remotes.json
WARN: imguizmo/cci.20210223: requirement imgui/1.82 overridden by your conanfile to imgui/cci.20220621+1.88.docking 
ERROR: clipper/4.10.0: Error in source() method, line 50
	tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
	NotFoundException: Not found: https://sourceforge.net/projects/polyclipping/files/Older%20versions/clipper_ver4.10.0.zip
```

Windows:  

```
WARN: Remotes registry file missing, creating default one in C:\Users\runneradmin\.conan\remotes.json
WARN: imguizmo/cci.20210223: requirement imgui/1.82 overridden by your conanfile to imgui/cci.20220621+1.88.docking 
-- Compiler is not EMSCRIPTEN
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


ERROR: clipper/4.10.0: Error in source() method, line 50
	tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
	NotFoundException: Not found: https://sourceforge.net/projects/polyclipping/files/Older%20versions/clipper_ver4.10.0.zip
```

Full workflow log files:  

- Ubuntu: [logs_1892.zip](https://github.com/conan-io/conan-center-index/files/9267351/logs_1892.zip).
- Windows: [logs_1893.zip](https://github.com/conan-io/conan-center-index/files/9267545/logs_1893.zip).

</details>

### PR DESCRIPTION
**Pull request by marcusl**
Older source versions are removed from Sourceforge's archive, so previous link was no longer valid, see #12064

I've recreated the archive, the original author restored it to the original location so we just need to update the checksum now.

 **clipper/4.10.0**

fixes #12064

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by marcusl** [comment]
First time contributing, so I'm trying to figure out how to create/test. Will undraft once tested.

**Review by jgsogo — CHANGES_REQUESTED** [review]
We only get sources from official repositories. If those sources are now missing, we can upload them to our internal Artifactory, as we did with `gsoap` (https://github.com/conan-io/conan-center-index/blob/master/recipes/gsoap/all/conandata.yml)

**Comment by marcusl** [comment]
If anyone is interested, the error I got when building 5.1.6 with `conan create . clipper/5.1.6@demo/testing` in recipes/clipper/all was:

```
clipper/5.1.6@demo/testing: Calling build()
WARN: patches/0001-include-install-directory-5.x.patch: file 1/1:        b'cpp/CMakeLists.txt'
WARN: patches/0001-include-install-directory-5.x.patch:  hunk no.1 doesn't match source file at line 11
WARN: patches/0001-include-install-directory-5.x.patch:   expected: b'# The header name clipper.hpp is too generic, so install in a subdirectory'
WARN: patches/0001-include-install-directory-5.x.patch:   actual  : b'SET(BUILD_SHARED_LIBS ON CACHE BOOL'
patches/0001-include-install-directory-5.x.patch: source file is different - b'cpp/CMakeLists.txt'
clipper/5.1.6@demo/testing:
clipper/5.1.6@demo/testing: ERROR: Package 'f26537b056c9ff213dcc2874372b233408a3e04e' build failed
clipper/5.1.6@demo/testing: WARN: Build folder C:\Users\marcu\.conan\data\clipper\5.1.6\demo\testing\build\f26537b056c9ff213dcc2874372b233408a3e04e
ERROR: clipper/5.1.6@demo/testing: Error in build() method, line 63
        tools.patch(**patch)
        ConanException: Failed to apply patch: patches/0001-include-install-directory-5.x.patch
```

**Comment by uilianries** [comment]
@marcusl patch is incompatible. Usually, it's end-of-file: CRLF vs LF. Otherwise, the patch is really incompatible.

**Comment by marcusl** [comment]
> @marcusl patch is incompatible. Usually, it's end-of-file: CRLF vs LF. Otherwise, the patch is really incompatible.

Yeah. I'm thinklng to skip that part as I didn't need it, so someone who needs 5.6.1 can find the right source that matches the patch or fix the patch or what not, since the original zip that worked for both 4.10.0 and 5.6.1 is lost (see the linked issue's discussion)

Anyway, I'm closing this as the zip file has been restored to sourceforge, but from my little archive repo. So I made some contribution at last. :)

**Comment by marcusl** [comment]
Need to update the SHA256. Reopening.

**Comment by jgsogo** [comment]
We will be adding new contributors in the following days and your PR will be triggered again 😸

**Comment by uilianries** [comment]
Okay, now I remember it's totally related to https://github.com/conan-io/conan-center-index/issues/12064

**Comment by marcusl** [comment]
It's just a sha change now.

**Review by uilianries — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Comment by marcusl** [comment]
@jgsogo pokepoke! :)

**Comment by prince-chrismc** [comment]
Can you please update the title so it matches the changes?

"clipper: update checksum sha256" or something similar please 🙏

**Review by jgsogo — APPROVED** [review]

**Comment by marcusl** [comment]
@prince-chrismc It doesn't seem as if your approval counted?

**Comment by jgsogo** [comment]
> @prince-chrismc It doesn't seem as if your approval counted?

It does count, but I was blocking the merge. If some member of the team requests changes in a PR, it is blocked regardless of other approvals or changes (this logic is managed by the bot that is performing the actual merges)

**Comment by marcusl** [comment]
> > @prince-chrismc It doesn't seem as if your approval counted?
> 
> It does count, but I was blocking the merge. If some member of the team requests changes in a PR, it is blocked regardless of other approvals or changes (this logic is managed by the bot that is performing the actual merges)

I get that your changed-requested blockedf things. However, now it requires 3 approvals with write access, but github says there are only two, and @prince-chrismc 's marks are green as yours and @uilianries are.

**Comment by marcusl** [comment]
Huh, now it passed. Yay. :)

**Comment by marcusl** [comment]
Thanks guys! :)

**Comment by jgsogo** [comment]
> > > @prince-chrismc It doesn't seem as if your approval counted?
> > 
> > 
> > It does count, but I was blocking the merge. If some member of the team requests changes in a PR, it is blocked regardless of other approvals or changes (this logic is managed by the bot that is performing the actual merges)
> 
> I get that your changed-requested blocked things. However, now it requires 3 approvals with write access, but github says there are only two, and @prince-chrismc 's marks are green as yours and @uilianries are.

(Just trying to explain the process)

The green/grey for the approvals are from Github and it depends on the permissions people have inside the Conan organization and this repo. We have also configured the repository to require 3 approvals with write access before merging a PR, but this is just to prevent ourselves from accidentally merging a PR manually. 

image


Our bot only takes into account what is written in this file (https://github.com/conan-io/conan-center-index/blob/master/.c3i/reviewers.yml), two approvals from people in that list are required, at least one of them from a team member. The bot has elevated permissions so it can merge PRs even if Github still says that "3 approvals with write permissions are required".

**Comment by marcusl** [comment]
Ah. Got it. Thanks for the detailed explanation, and for conan in general. It's 100% awesomesauce! 🤩

**Comment by prince-chrismc** [comment]
👋 doc writer here...

We have docs here https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#automatic-merges
An I am curious if you looked for them or just relied on GitHub status?

(trying to see what improvements i could make 🙏 )

**Comment by marcusl** [comment]
I had not seen that. I scanned the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) as mentioned/linked in the issue template though. 

Maybe link to the review process doc from the guide or as a separate note in the issue template?

**Comment by prince-chrismc** [comment]
Thanks! those are good suggestion 👍 
It's linked, it's the 3️⃣ but it't not clear enough... template sounds like a sure win

I appreciate the feedback!

**Pull request merged**
